sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -18,7 +18,7 @@
   private @Nullable Long appStartEndMillis;
 
   /** The type of App start coldStart=true -> Cold start, coldStart=false -> Warm start */
-  private boolean coldStart;
+  private @Nullable Boolean coldStart = null;
```

Comment:
we kinda have 3 states now for this: unset, cold, warm - should we introduce an enum rather?

Reply:
yep, `enum` is more expensive but it makes sense, since this is internal, I will make it in another PR to unblock this one and cut a release